### PR TITLE
FAQ: scan_tips

### PR DESCRIPTION
**Pull request by matriceria**
Due to issue #2093, I added these two entries to the FAQ section.
Screenshot 2021-11-18 at 10 09 31
Screenshot 2021-11-18 at 10 10 02

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -883,6 +883,20 @@
                             "This screen is shown on iPhones running an iOS version between iOS 13.5 and (including) iOS 13.6.1. On these iOS Versions, the necessary version 2 of the Exposure Notification Framework is not available. Therefore you need to update to iOS 13.7 or newer, to continue to use the Corona-Warn-App. More information can be found in <a href='#ios135' target='_blank' rel='noopener noreferrer'>this</a> FAQ entry."
                         ]
                     },
+                    {
+                        "title": ["Why does the CWA not scan a QR code?",
+                                "  Why does the QR code scanner not respond?"
```
Comment (MikeMcC399) [inline src/data/faq.json:888]:
```suggestion
                                "Why does the QR code scanner not respond?"
```
remove leading space

File: src/data/faq_de.json [hunk 2]
```diff
@@ -883,6 +883,20 @@
                             "Dieser Bildschirm wird auf iPhones mit einer iOS-Version zwischen iOS 13.5 und (einschließlich) iOS 13.6.1 angezeigt. Auf diesen iOS-Versionen ist die benötigte Version 2 des Exposure Notification Frameworks nicht verfügbar. Deshalb muss erst auf iOS 13.7 oder neuer aktualisiert werden, um die Corona-Warn-App weiterhin verwenden zu können. Weitere Informationen finden Sie in <a href='#ios135' target='_blank' rel='noopener noreferrer'>diesem</a> FAQ Eintrag."
                         ]
                     },
+                    {
+                        "title": ["Warum wird der QR-Code von der CWA nicht gescannt?",
+                                "  Warum reagiert der QR-Code-Scanner nicht?"
```
Comment (MikeMcC399) [inline src/data/faq_de.json:888]:
```suggestion
                                "Warum reagiert der QR-Code-Scanner nicht?"
```

File: src/data/faq_de.json [hunk 3]
```diff
@@ -883,6 +883,20 @@
...
                         ]
                     },
+                    {
+                        "title": ["Warum wird der QR-Code von der CWA nicht gescannt?",
+                                "  Warum reagiert der QR-Code-Scanner nicht?"
+                            ],
+                        "anchor":"scan_tips",
+                        "active":true,
+                        "textblock":[
+                            "Auf einigen Geräten erkennt die Kamera den QR-Code nicht sofort und es passiert zunächst gar nichts, d.h. es wird weder der QR-Code eingelesen, noch eine Fehlermeldung angezeigt. Dieses Verhalten betrifft vorrangig einige Android-Geräte, kann jedoch auch auf iOS-Geräten auftreten.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:893]:
```suggestion
                            "Auf einigen Geräten erkennt die Kamera den QR-Code nicht sofort und es passiert zunächst gar nichts, d.&nbsp;h. es wird weder der QR-Code eingelesen, noch eine Fehlermeldung angezeigt. Dieses Verhalten betrifft vorrangig einige Android-Geräte, kann jedoch auch auf iOS-Geräten auftreten.",
```

In German `&nbsp;` is needed in abbreviations. See also https://github.com/corona-warn-app/cwa-website/issues/1689.

File: src/data/faq_de.json [hunk 4]
```diff
@@ -883,6 +883,20 @@
...
+
+                            "Folgende Tipps helfen oft dabei, den QR-Code zu scannen:",
+                            "<ul> <li> Auf dem QR-Code erkennen Sie 3 größere Quadrate in der Ecke: Diese müssen vollständig von der Kamera erfasst werden.</li> <li>Halten Sie die Kamera etwas weiter vom QR-Code weg, bis er in etwa die Größe einer Briefmarke hat.</li> <li>Sorgen Sie für ausreichende Kontraste und Lichtverhältnisse.</li> <li>Drehen Sie den QR-Codes um 90 Grad und versuchen Sie es erneut.</li></ul>",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:896]:
```suggestion
                            "<ul> <li> Auf dem QR-Code erkennen Sie 3 größere Quadrate in der Ecke: Diese müssen vollständig von der Kamera erfasst werden.</li> <li>Halten Sie die Kamera etwas weiter vom QR-Code weg, bis er in etwa die Größe einer Briefmarke hat.</li> <li>Sorgen Sie für ausreichende Kontraste und Lichtverhältnisse.</li> <li>Drehen Sie den QR-Code um 90 Grad und versuchen Sie es erneut.</li></ul>",
```
Code should be singular in last sentence.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
In-line suggestions added.

**Comment by MikeMcC399** [comment]
@matriceria 
The syntax of the title is wrong and it stops other articles being displayed. I suggest to put the title on one line.

Look at the other entries to compare how they are written.

File: src/data/faq_de.json [hunk 5]
```diff
@@ -883,6 +883,20 @@
                             "Dieser Bildschirm wird auf iPhones mit einer iOS-Version zwischen iOS 13.5 und (einschließlich) iOS 13.6.1 angezeigt. Auf diesen iOS-Versionen ist die benötigte Version 2 des Exposure Notification Frameworks nicht verfügbar. Deshalb muss erst auf iOS 13.7 oder neuer aktualisiert werden, um die Corona-Warn-App weiterhin verwenden zu können. Weitere Informationen finden Sie in <a href='#ios135' target='_blank' rel='noopener noreferrer'>diesem</a> FAQ Eintrag."
                         ]
                     },
+                    {
+                        "title": ["Warum wird der QR-Code von der CWA nicht gescannt?",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:887]: Wrong syntax for title!

File: src/data/faq.json [hunk 6]
```diff
@@ -883,6 +883,20 @@
                             "This screen is shown on iPhones running an iOS version between iOS 13.5 and (including) iOS 13.6.1. On these iOS Versions, the necessary version 2 of the Exposure Notification Framework is not available. Therefore you need to update to iOS 13.7 or newer, to continue to use the Corona-Warn-App. More information can be found in <a href='#ios135' target='_blank' rel='noopener noreferrer'>this</a> FAQ entry."
                         ]
                     },
+                    {
+                        "title": ["Why does the CWA not scan a QR code?",
```
Comment (MikeMcC399) [inline src/data/faq.json:887]: Please check this syntax.

**Comment by matriceria** [comment]
@MikeMcC399 Thanks for the feedback and the corrections!

File: src/data/faq_de.json (hunk 5 above)
Comment (dsarkar) [reply src/data/faq_de.json]: corrected, thanks

File: src/data/faq.json (hunk 6 above)
Comment (dsarkar) [reply src/data/faq.json]: corrected, thanks

File: src/data/faq_de.json (hunk 4 above)
Comment (dsarkar) [reply src/data/faq_de.json]: corrected, thanks

File: src/data/faq_de.json (hunk 3 above)
Comment (dsarkar) [reply src/data/faq_de.json]: corrected, thanks

File: src/data/faq.json [hunk 7]
```diff
@@ -884,9 +884,7 @@
                         ]
                     },
                     {
-                        "title": ["Why does the CWA not scan a QR code?",
-                                "  Why does the QR code scanner not respond?"
-                            ],
+                        "title":"Why does the CWA not scan a QR code?, Why does the QR code scanner not respond?",
```
Comment (MikeMcC399) [inline src/data/faq.json]:
```suggestion
                        "title": "Why does the CWA not scan a QR code? Why does the QR code scanner not respond?",
```
Remove comma, so it is like other titles with multiple sentences.

File: src/data/faq_de.json [hunk 8]
```diff
@@ -884,16 +884,14 @@
                         ]
                     },
                     {
-                        "title": ["Warum wird der QR-Code von der CWA nicht gescannt?",
-                                "  Warum reagiert der QR-Code-Scanner nicht?"
-                            ],
+                        "title": "Warum wird der QR-Code von der CWA nicht gescannt?, Warum reagiert der QR-Code-Scanner nicht?",
```
Comment (MikeMcC399) [inline src/data/faq_de.json]:
```suggestion
                        "title": "Warum wird der QR-Code von der CWA nicht gescannt? Warum reagiert der QR-Code-Scanner nicht?",
```

Remove comma to match other titles with multiple sentences.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
It would be good if you could format the new sections the standard way for json.

Do you use Visual Studio Code? If yes, then just select the new text and use:
`Ctrl+K Ctrl+F Format selection `

**Comment by MikeMcC399** [comment]
Apart from that the content is now good. I also made a suggestion to remove the commas in the titles.

File: src/data/faq.json (hunk 7 above)
Comment (dsarkar) [reply src/data/faq.json]: yep, thanks

File: src/data/faq_de.json (hunk 8 above)
Comment (dsarkar) [reply src/data/faq_de.json]: yep, thanks

**Review by MikeMcC399 — APPROVED** [review]
Visual Studio Code would add a space in
`"textblock":[`
to make it
`"textblock": [`

however that is a very minor point and no reason to delay a merge of this PR.

**Comment by Ein-Tim** [comment]
Can the title of this PR be changed to something more meaningful, please?

**Pull request merged**